AudioGenerator.py

```diff
@@ -0,0 +1,181 @@
+import math, wave, struct, os, random
+
+
+numChannels = 1                                                        # Decides whether your in mono or stereo
+audioHz = 44100                                                        # Sample rate
+maxAmp = 32767                                                         # 32767.0 is the max amp for 16bit audio
+amplitude = 1
+
+#makes a list of sin values to be converted later
+def makeSin(audioHz, frequency, numSample, maxAmp, amplitude):
```

Comment:
Perhaps a docstring could be included here to elaborate on what the arguments for this method should be and their types. 
And/or the variables could be named more clearly - for example, I'm not sure what "audioHz" is supposed to represent.